### DISCUSSION TITLE
The Twain Protocol or Scanning support / Magick.NET

### DISCUSSION THREAD

**Discussion by MrZakaz** [ImageMagick/ImageMagick#3400] [General]
Hello, I would like to know if Magick.NET has the Twain protocol or some scanning support - And if not what the authors suggest it supports?
What can I do for my SkanPort with Magick.NET to be able to download an image from the scanner?

**Comment by snibgo** [comment]
You have already asked this at https://github.com/ImageMagick/ImageMagick/discussions/3391

Please don't create new threads with the same question.

**Reply by MrZakaz** [reply]
Then is image magick the same as magick.net?
I mean, will it also work in magick.net?

**Comment by snibgo** [comment]
If the "SCAN:" pseudo-format works for any interface to ImageMagick, it will work for all interfaces. Use "-list format" to see if the format is supported on your build.

I don't have a build of IM that supports "SCAN:".

**Reply by MrZakaz** [reply]
Thanks !
